app/helpers/users_helper.rb

```diff
@@ -9,8 +9,10 @@ def user_role_options
   end
 
   def user_lock_status(user)
-    if user.access_locked?
-      'Locked'
+    if user.disabled_by_fund?
+      'Locked by admin'
+    elsif user.locked_at?
```

Comment:
Why is this now `locked_at?` but before it was `access_locked?`?

Reply:
Turns out `access_locked?` is one of those things we get for free from devise, so I restored this.